src/System.Text.Encoding/tests/Performance/Perf.Encoding.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+using Xunit;
+using Microsoft.Xunit.Performance;
+
+namespace System.Runtime.Tests
+{
+    public class Perf_Encoding
+    {
+        [Benchmark]
+        [InlineData(40)]
+        [InlineData(160)]
+        public void GetBytes_str(int size)
+        {
+            Encoding enc = Encoding.UTF8;
+            string toEncode = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    enc.GetBytes(toEncode); enc.GetBytes(toEncode); enc.GetBytes(toEncode);
+                    enc.GetBytes(toEncode); enc.GetBytes(toEncode); enc.GetBytes(toEncode);
+                    enc.GetBytes(toEncode); enc.GetBytes(toEncode); enc.GetBytes(toEncode);
```

Comment:
Another example where the choice of strings could impact perf, e.g. 'a'-'f' and '0'-'9' are all ASCII.